core/node/libp2p/routingopt.go

```diff
@@ -23,6 +24,64 @@ type RoutingOption func(
 	...peer.AddrInfo,
 ) (routing.Routing, error)
 
+// Default HTTP routers used in parallel to DHT when Routing.Type = "auto"
+var defaultHTTPRouters = []string{
+	"https://dev.cid.contact/routing/v1", // TODO: remove when cid.contact is ready
+	"https://cid.contact/routing/v1",     // https://github.com/ipfs/kubo/issues/9422#issuecomment-1338142084
```

Comment:
@willscott @ajnavarro should we ship both in 0.18.0-rc1? Switch to cid.contact?

Reply:
Ok, removed `dev.cid.contact` – https://github.com/ipfs/kubo/pull/9475/commits/5396526ce2bb73cafc257067b654fd998698511d